server/app/com/xsn/explorer/data/LedgerDataHandler.scala

```diff
@@ -21,7 +21,8 @@ trait LedgerDataHandler[F[_]] {
   def push(
       block: Block.HasTransactions,
       tposContracts: List[TPoSContract],
-      filterFactory: () => GolombCodedSet
+      filterFactory: () => GolombCodedSet,
+      rewards: Option[BlockRewards]
```

Comment:
I believe that all blocks have rewards.

Historically, the explorer wasn't able to retrieve the genesis transaction due to some weird behavior from bitcoind/xsnd, if you get the block and then the transactions, the genesis transaction can't be retrieved, if you retrieve the full block instead, it will contain the genesis transaction

So, the right thing to do is to fix the block rewards computation to work with the genesis block, which is a little bit tricky, the better way that I can think right now is to update the `XSNService.{getTransaction, getRawTransaction}` to detect when we are looking for the genesis transaction, in which case, it will be retrieved from the fullblock. The config already has the genesis block and we could include the genesis transaction.